api/migrations/2020-11-08-174607_trigger_users/up.sql

```diff
@@ -0,0 +1,58 @@
+CREATE INDEX user_program_records_user_program_ids ON user_program_records (program_spec_id, user_id);
```

Comment:
this name doesn't really seem applicable, the index doesn't include `user_programs`